Lib/fontbakery/checkrunner.py

```diff
@@ -278,11 +279,16 @@ def __init__(self, profile, values
                                        f'{message}')
         self._values = values
 
+        self.use_cache = use_cache
         self._cache = {
             'conditions': {}
           , 'order': None
         }
 
+    def clearCache(self):
```

Comment:
this new method as well as `self.use_cache` do not seem related to the new testing helpers

Reply:
Yes, I thought I'd need these when I started this, but since I throw away the check-runner instance after one time use, there's no actual need for a cache reset.